index.html

```diff
@@ -137,66 +173,101 @@ <h2 class="cont-header">Professional Art Printing Data</h2>
-                    <h3 class="iconic"></h3>Skills</div>
-                <ul class="langlist">
-                    <li class ="jss">Codekit</li>
-                    <li class="jss">Github</li>
-                    <li class="jss">Codepen</li>
-                    <li class="jss"> Gitlab</li>
-                    <li class="jss"> Terminal</li>
-                </ul>
+
+            <div class="underline"></div>
+            <div class="lang-cont">
+                <div class="language-container">
+                    <div class="lang-heading"><img src="images/rhombus.png" alt="">
+                        <h3 class="iconic">Language</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Javascript</li>
+                        <li class="jss">Ruby</li>
+                        <li class="jss">Html</li>
+                        <li class="jss"> Css</li>
+                    </ul>
+                </div>
+                <div class="framecont">
+                    <div class="lang-heading"><img src="images/rectangle.png" alt="">
+                        <h3 class="iconic">Framework</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Bootstrap</li>
+                        <li class="jss">Ruby on Rails</li>
+                        <li class="jss">RSPec</li>
+                        <li class="jss">CapyBara</li>
+                        <li class="jss">Selenium</li>
+                    </ul>
+                </div>
+                <div class="skillscont">
+                    <div class="lang-heading"><img src="images/circle.png" alt="">
+                        <h3 class="iconic"></h3>Skills
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Codekit</li>
+                        <li class="jss">Github</li>
+                        <li class="jss">Codepen</li>
+                        <li class="jss"> Gitlab</li>
+                        <li class="jss"> Terminal</li>
+                    </ul>
+                </div>
             </div>
-        </div>
 
     </section>
 
+    <form action="https://formspree.io/f/mlevdeoa" method="post" class="contact-form">
+        <div class="interest">
+            <h5>I'm always interested in hearing about new projects,
+                so if you'd like to chat please get in touch.</h5>
+        </div>
+        <div class="all">
+            <div><input type="text" placeholder="Full name" maxlength="30" class="inputs" name="user-name" required>
+            </div>
+            <div><input type="email" placeholder="Email" class="inputs" name="e-mail" required></div>
+            <div class="text-cont"><textarea name="message" placeholder="Enter your text here" maxlength="500"
+                    class="ta" required></textarea> </div>
+            <button type="submit" id="get-me" required class="btz">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] Please try to work on the styles of this submit button in order to match the styles shown in the Figma design. Take a look at the following comparison 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/186246541-75e2c4f1-fe8b-4334-b1a5-1e25db35902b.png)

**Figma**
![image](https://user-images.githubusercontent.com/90222110/186246752-88a60c8c-1f18-4ea9-ab3c-ec295cc2c32f.png)